Makefile.am

```diff
@@ -5657,17 +5657,7 @@ dist_noinst_DATA += \
     contrib/sssd.spec.in \
     BUILD.txt \
     COPYING \
-    src/tests/multihost/README.md \
-    src/tests/multihost/conftest.py \
-    src/tests/multihost/basic/mhc.yaml \
-    src/tests/multihost/basic/test_basic.py \
-    src/tests/multihost/basic/test_config.py \
```

Comment:
Hi,

do I see correctly that you added `src/tests/multihost/basic/test_config.py` to the backport? What's the reason, looks like the file is still present in the source tree?

bye,
Sumit

Reply:
But original patch - 4b2553d42fe063c2130e6f972508104a710cc811 - simply replaced everything under 'src/tests/*' with single `src/tests/`

Do I misunderstand something?

Reply:
This is a `dist_noinst_DATA` , not a list of tests to run.

Reply:
Hi,

I just wanted to make sure that this was intentional. In the meantime I realize that `test_config.py` was removed in master already individually by 4a44cca401dbec16593b93b5c4333005cf6165f4.

bye,
Sumit